storage/src/device.rs

```diff
@@ -448,15 +457,27 @@ impl BlobChunkInfo for BlobIoChunk {
     }
 
     fn id(&self) -> u32 {
-        self.as_base().id()
+        // BlobIoChunk::Address is a medium type to pass chunk IO description
+        // for rafs v6. It can't implement BlobChunkInfo and calling `as_base`
+        // causes panic. So this is a workaround to avoid panic.
```

Comment:
Wouldn't it panic too in all the other methods that also calls `self.as_base()`?

Reply:
Yes. other methods are also calling `as_base()`, however, they are not required and called by `BlobIoDesc()`. So they will not cause program panic now. My thought is to refactor the code a bit in the future in Rafs layer and to solve the panic problem for once and all.